### PR TITLE
update czech translation

### PR DESCRIPTION
**Pull request by zen0bit**
Definetly not final...

### CODE REVIEW

File: po/cs.po [hunk 1]
```diff
@@ -833,15 +830,16 @@ msgstr ""
 "tabelátoru"
 
 #: data/geany.glade:3348
-#, fuzzy
 msgid "_Backspace key unindents"
-msgstr "Klávesa tabelátor odsazuje"
+msgstr "_Klávesa tabelátor odsazuje"
```
Comment (andrej-herceg) [inline po/cs.po:834]: backspace here?

Comment (zen0bit) [reply po/cs.po]: done

**Comment by frlan** [comment]
Hi, 

Why are you removing a bigger number of punctuation?

**Comment by zen0bit** [comment]
just to be same as original (poedit complain)

**Pull request merged**